### ISSUE TITLE
Add abstract-blob-store as a dependency

### ISSUE THREAD

**Issue by clintonb** [feathersjs-ecosystem/feathers-blob#92]
See #81. If `abstract-blob-store` is needed to compile the project, `abstract-blob-store` should be a production dependency. It's currently a development dependency, so not installed when used in other projects.

**Comment by claustres** [comment]
I am not a typescript expert but as far as I can see a dependence has been added in the types definition. However, from the JS code perspective this dependency is only used in tests (so as a dev dependency). @daffl probably knows how to fix this.

**Comment by daffl** [comment]
I'd say that everything that is required in the production types file should be a production dependency. An alternative (since `abstract-blob-store` is really just a test harness) would be to add the Abstract Blob store types from https://github.com/maxogden/abstract-blob-store/blob/master/index.d.ts directly in the typings file here.

**Comment by claustres** [comment]
I agree, I am not a typescript fan but IMHO using it implies that we should depend on the interface so that if it is changed it will break at compilation time.

**Comment by daffl** [comment]
Moving `abstract-blob-store` from `devDependencies` into `dependencies` should do it then.
